pages/en/hosted-service/migrating-subgraph.mdx

```diff
@@ -64,6 +72,8 @@ graph codegen && graph build
 
 And that's it! After you are done publishing, you'll be able to view your subgraphs live on the network via [The Graph Explorer](https://thegraph.com/explorer).
 
+Feel free to leverage the [#Curators channel](https://discord.gg/rC8rBuRtbH) on Discord to let Curators know that your subgraph is ready to be signaled. It would also be helpful if you share your expected query volume with them.
```

Comment:
Nice! Maybe works best to clarify how this information helps (maybe "Help curators by giving them some context about your subgraph, and its expected query volume")